### ISSUE TITLE
1.18.2 server crashes after 2nd server start

### ISSUE THREAD

**Issue by dadoirie** [CleverNucleus/Data-Attributes#21]
Basically if creating a new world everything works fine, but after restarting the server it crashes and if restarting without data attributes (and same world) it stops from crashing.
It's surely a conflict which I'm currently investigating by simply disabling mods one by one after each restart until it is not crashing anymore ... not even sure if that will even work without deleting the world on which is it crashing, because the issue is a ticking entity (at least that's what shared the console log says starting at line 2060)
[https://mclo.gs/NdpO2xr](https://mclo.gs/NdpO2xr)

what could it be? 

EDIT: and nope tctc it isn't as I've disabled that one as first (because it is part of the error)

**Comment by dadoirie** [comment]
Found it ... it crashes with the graveyard mod.
As mentioned it doesn't crash upon world creation and keeps crashing after the first server restart after finishing loading the world ... after `Loading Xaero's World Map - Stage 2/2 (Server)` which is basically the last of the server start up

**Comment by CleverNucleus** [comment]
Hi there,
I will look into it.
Thanks for bringing this to my attention.

**Comment by dadoirie** [comment]
But it might be graveyard's fault entirely, because this mod had some crashes in a previous attempt making this modpack (which I have deleted that MC instance, but I remember removing graveyard for that) when the entities got loaded
something might be wrong with the entities of that mod and might need a rewrite itself

**Comment by CleverNucleus** [comment]
Hi there,
I have identified the problem and indeed it is with the Graveyard mod. I have since fixed the issue and created a pull request for them to merge and release with on their end - it can be viewed [here](https://github.com/finallion/The_Graveyard_-FABRIC-/pull/41). The changes I have brought forward should also be applied to releases of the mod on Minecraft 1.19+, as the incompatibility persists there too - I have explained this so the author is aware.

Thanks for bringing this to my attention. I will close this issue now as it is up to the Graveyard mod's author to implement the fix I have provided.
